src/utils/trust/legal.entity.beneficial.owner.mapper.ts

```diff
@@ -34,10 +34,9 @@ const mapLegalEntityToSession = (
     identification_legal_form: formData.legalForm,
     is_service_address_same_as_principal_address: (formData.is_service_address_same_as_principal_address) ? Number(formData.is_service_address_same_as_principal_address) : 0,
     is_on_register_in_country_formed_in: (formData.is_on_register_in_country_formed_in) ? Number(formData.is_on_register_in_country_formed_in) : 0,
-    is_corporate_still_involved_in_trust: formData.stillInvolved,
-    ceased_date_day: formData.stillInvolved === "0" ? formData.ceasedDateDay : "",
-    ceased_date_month: formData.stillInvolved === "0" ? formData.ceasedDateMonth : "",
-    ceased_date_year: formData.stillInvolved === "0" ? formData.ceasedDateYear : "",
+    ceased_date_day: formData.stillInvolved === "No" ? formData.ceasedDateDay : "",
```

Comment:
i'm pretty sure the formData.stillInvolved values will be 1's and 0's as the form data is the values submitted by the html form (which uses 1's and 0's). so i think the ceased date conversions were correct before

Reply:
Yes, seems to be in conflict with line 60 where you do this check: `formData.stillInvolved === "1"`

Reply:
Changed this back.